Source/WebCore/platform/graphics/cocoa/WebCoreDecompressionSession.h

```diff
@@ -91,6 +92,8 @@ class WEBCORE_EXPORT WebCoreDecompressionSession : public ThreadSafeRefCountedAn
     bool hardwareDecoderEnabled() const { return m_hardwareDecoderEnabled; }
     void setHardwareDecoderEnabled(bool enabled) { m_hardwareDecoderEnabled = enabled; }
 
+    void setResourceOwner(const ProcessIdentity& resourceOwner) { m_resourceOwner = resourceOwner; }
```

Comment:
m_resourceOwner is set on the main thread but read on the decompressionQueue. 

we need an annotation `WTF_GUARDED_BY_CAPABILITY(m_decompressionQueue.get());` or that it's accessed under a lock 

but first of all , Is the ProcessIdentity thread-safe  ? I'm guessing a task to the main thread calling `setOwnershipIdentity` is a safer approach